pages/api/mech-metrics.js

```diff
@@ -91,6 +92,92 @@ const fetchMechRequestsFromSubgraphs = async () => {
   }
 };
 
+// Classified totals derived from subgraphs (Mech + Mech-Marketplace Gnosis + Base)
+const fetchCategorizedRequestTotals = async () => {
+  // Agent IDs by category (from SQL classification)
+  const predictTraderIds = [14, 25, 13];
+  const contributeIds = [6];
+  const governatooorIds = [5];
```

Comment:
please let's move to constants similar to PREDICT_AGENT_CLASSIFICATION, and reuse in components/DataPage/MechCategorizedRequests.jsx